signUp.html

```diff
@@ -0,0 +1,83 @@
+<html>
```

Comment:
Always prettify your file for better readability. There are lots of tools online!